integrations/aws/main.py

```diff
@@ -35,6 +35,10 @@
     is_access_denied_exception,
     is_server_error,
 )
+from port_ocean.utils.async_iterators import stream_async_iterators_tasks
+import asyncio
+
+MAX_CONCURRENT_TASKS = 50
```

Comment:
Why is this the limit?

Reply:
Controlling the number of accounts with the stream at a particular time ?

Reply:
I meant, Why did you pick 50 for the max concurrent tasks? is there a doc you are basing on? if yes - let's add it as comment

Reply:
Right, I understand, actually @Tankilevitch  and I decided on this value intuitively control the stream

Reply:
what do you think ?

Reply:
Ok, understood.